kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchResultAggregator.java

```diff
@@ -6,4 +6,6 @@
 public interface SearchResultAggregator<T> {
   CompletableFuture<SearchResult<T>> aggregate(
       CompletableFuture<List<SearchResult<T>>> searchResults);
+
+  SearchResult<T> aggregate(List<SearchResult<T>> searchResults);
```

Comment:
Please add a comment on when we need to use one over the other.

Reply:
fixed - removed it so we don't have two method :)